actionpack/test/controller/routing_test.rb

```diff
@@ -334,10 +334,9 @@ def test_ignores_leading_slash
   end
 
   def test_route_with_colon_first
-    rs.draw do
-      get '/:controller/:action/:id', :action => 'index', :id => nil
-      get ':url', :controller => 'tiny_url', :action => 'translate'
-    end
```

Comment:
why do we have to change the setup like this? I'd rather keep the existing setup than potentially changing the tests meaning.